parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -99,12 +119,11 @@ $global:KubeletNodeLabels = "{{GetAgentKubernetesLabelsDeprecated . }}"
 {{end}}
 $global:KubeletConfigArgs = @( {{GetKubeletConfigKeyValsPsh .KubernetesConfig }} )
 
-$global:UseManagedIdentityExtension = "{{GetVariable "useManagedIdentityExtension"}}"
-$global:UserAssignedClientID = "{{GetVariable "userAssignedClientID"}}"
-$global:UseInstanceMetadata = "{{GetVariable "useInstanceMetadata"}}"
+$global:UseManagedIdentityExtension = "{{GetVariableProperty "customData" "useManagedIdentityExtension"}}"
+$global:UseInstanceMetadata = "{{GetVariableProperty "customData" "useInstanceMetadata"}}"
 
-$global:LoadBalancerSku = "{{GetVariable "loadBalancerSku"}}"
-$global:ExcludeMasterFromStandardLB = "{{GetVariable "excludeMasterFromStandardLB"}}"
+$global:LoadBalancerSku = "{{GetVariableProperty "customData" "loadBalancerSku"}}"
+$global:ExcludeMasterFromStandardLB = "{{GetVariableProperty "customData" "excludeMasterFromStandardLB"}}"

```

Comment:
this is also not needed for AKS

Reply:
@xuto2 I think that removing these  code should be in another PR if they are unnecessary and we only keep the minimal changes in this PR. It will reduce the risk for such a complicated change.

Reply:
`excludeMasterFromStandardLB` is set to true by default in Kuberentes,  should we also remove this from Linux?
https://github.com/kubernetes/kubernetes/pull/61884

Reply:
my bad. I confused myself with a different property in aks-engine code. Please keep it.
sorry for the confusion.